### PR TITLE
21028 Disabled AR todo item for non-BEN corps

### PR DESCRIPTION
**Pull request by severinbeauvais**
*Issue #:* bcgov/entity#21028

*Description of changes:*
- app version = 7.2.8
- disabled AR todo item is COLIN FF is disabled
- deleted an unused getter

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the business-filings-ui license (Apache 2.0).

### CODE REVIEW

File: src/components/Dashboard/Alerts/CorporateOnline.vue [hunk 1]
```diff
@@ -56,7 +56,6 @@ export default class CorporateOnline extends Vue {
 
   @Getter(useConfigurationStore) getCorporateOnlineUrl!: string
   @Getter(useBusinessStore) getLegalType!: CorpTypeCd
-  @Getter(useBusinessStore) isDisableNonBenCorps!: boolean
```
Comment (severinbeauvais) [inline src/components/Dashboard/Alerts/CorporateOnline.vue:59]: Not used in this component.

File: src/components/Dashboard/TodoList.vue [hunk 2]
```diff
@@ -951,7 +952,7 @@ export default class TodoList extends Mixins(AllowableActionsMixin, DateMixin, E
         arMinDate: header.arMinDate, // COOP only
         arMaxDate: header.arMaxDate, // COOP only
         status: header.status || FilingStatus.NEW,
-        enabled: task.enabled,
+        enabled: task.enabled && !this.isDisableNonBenCorps,
```
Comment (severinbeauvais) [inline src/components/Dashboard/TodoList.vue:955]: This is the method that loads the todo item for an Annual Report.

This code change sets `enabled` to False if the FF isn't set.

Screenshot when FF is True (ie, business is allowed).
image

Screenshot when FF is False (ie, business is not allowed).
image

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://business-filings-dev--pr-660-5c992kkj.web.app

**Review by ArwenQin — APPROVED** [review]
LGTM

**Pull request merged**